### DISCUSSION TITLE
Ways to Promote the Project

### DISCUSSION THREAD

**Discussion by blahguy88** [ThioJoe/YT-Spammer-Purge#1027] [General]
Clearly, the more people know about this project, the sooner the bots will be done away with. Problem is, I don't think enough people are using it, so I was wondering. Does anybody have any ideas how we could promote the project?

**Comment by ThioJoe** [comment]
It's already had mentions by some very large channels like [Linus Tech Tips](https://www.youtube.com/watch?v=zo_uoFI1WXM) and [MKBHD](https://www.youtube.com/watch?v=1Cw-vODp-8Y), so the word is definitely out. I know that even if creators don't necessarily know about it, a lot of fans will tell them about the program if they mention getting spam comments. This seems to happen on twitter pretty often.

**Reply by blahguy88** [reply]
I still don't think it's enough, because I'm not certain enough people even use it.

**Comment by RoboPterodactyl** [comment]
He could make an advertisement targeted toward creators

**Comment by Casondoesstuuffz** [comment]
Maybe we could get Other big channels to talk about it or dm creators and tell them about it (if that is a option)
